### PR TITLE
SLB-435: redirects

### PR DESCRIPTION
**Pull request by chindris**

### CODE REVIEW

File: apps/website/package.json [hunk 1]
```diff
@@ -27,14 +27,16 @@
...
     "vitest": "^1.1.1"
   },
   "scripts": {
     "test:static": "tsc --noEmit && eslint '**/*.{ts,tsx,js,jsx}' --ignore-path='./.gitignore'",
     "full-rebuild": "pnpm clean && pnpm build:waku",
     "start:drupal": "pnpm run --filter @custom/cms start",
     "build:drupal": "CLOUDINARY_CLOUDNAME=test pnpm start-test start:drupal 8888 build:waku",
-    "build:waku": "waku build",
+    "build:waku": "waku build && pnpm redirects",
+    "redirects": "tsx ./src/build/redirects.ts --dir=dist/public",
```
Comment (pmelab) [inline apps/website/package.json]: ULTRA NITPICK! 🤓 
Maybe call this `build:redirects` to make clear this is part of the build process.

File: apps/cms/config/sync/language/de/views.view.redirect.yml [hunk 2]
```diff
@@ -44,19 +44,19 @@ display:
             label: Statuscode
             group_items:
               1:
-                title: '300 Multiple Choices'
+                title: '300 Mehrere Auswahlmöglichkeiten'
```
Comment (pmelab) [inline apps/cms/config/sync/language/de/views.view.redirect.yml]: Drush export shenanigans? I would rather not include this in the diff. Lets keep it english by default.

**Review by pmelab — CHANGES_REQUESTED** [review]

File: apps/cms/config/sync/language/de/views.view.redirect.yml (hunk 2 above)
Comment (chindris) [reply apps/cms/config/sync/language/de/views.view.redirect.yml]: This should be removed now.

File: apps/website/package.json (hunk 1 above)
Comment (chindris) [reply apps/website/package.json]: This is now renamed, I also think it makes more sense to have it called "build:redirects"

**Review by pmelab — APPROVED** [review]

**Pull request merged**